config/config.go

```diff
@@ -66,9 +66,12 @@ func Load(file string) (config ProjectConfig, err error) {
 
 func fix(config ProjectConfig) ProjectConfig {
 	if len(config.Files) == 0 {
-		config.Files = []string{
-			"README.md",
-			"LICENSE.md",
+		config.Files = []string{}
+
+		for _, f := range []string{"README.md", "LICENCE.md", "LICENSE.md"} {
```

Comment:
Thats not a bad idea. I think the patterns should probably be abstracted out of that function, too; it'd be more maintainable in a var statement at the top.

I'll get something going tomorrow